alpine-rails-buildkite/etc/sv/buildkite

```diff
@@ -0,0 +1,13 @@
+#!/usr/bin/env bash
+
+set -o errexit -o pipefail -o noglob -o noclobber -o nounset
+IFS=$'\n\t'
+
+export HOME=/var/buildkite
+export GEM_HOME=${HOME}/.gem
```

Comment:
The only argument I have is that `${HOME}/` is easier to parse by eyes, but for now lets stick with `$HOME/`.